### ISSUE TITLE
Improve contrast ratio of some links when the design is ready

### ISSUE THREAD

**Issue by cindyli** [inclusive-design/wecount.inclusivedesign.ca#331]
## Describe the bug

Running lighthouse report on "Inclusion Challenges" page and News and Views post pages will report low contrast ratio on:

1. “Nomination form” and "contact form” links on “Inclusion Challenges” page. They are currently blue on white;
2. All tags (“AI”, “Inclusion” and “Smart Cities”) in the “Tags” sections on post pages. They are currently blue on orange;
3. All links in the main article on “Learn” page and "Initiatives" page. They are currently blue on white;

## To Reproduce

1. Go to these pages and run lighthouse report;
2. Check accessibility section for reported issues.

## Expected behavior

These links should have higher contrast ratio.

**Comment by Raunaksingh100** [comment]
@cindyli  @greatislander 

hi i am facing this problem when i add this COMMAND "NPM RUN START" plz anyone tell me how i run this project in my local plz help,
see in the screenshot for the better understanding....


![my issue with npm](https://user-images.githubusercontent.com/65535360/87951073-c161c680-cac5-11ea-83e0-91c0b5336fc0.PNG)

**Comment by cindyli** [comment]
> @cindyli @greatislander
> 
> hi i am facing this problem when i add this COMMAND "NPM RUN START" plz anyone tell me how i run this project in my local plz help,
> see in the screenshot for the better understanding....
> 
> ![my issue with npm](https://user-images.githubusercontent.com/65535360/87951073-c161c680-cac5-11ea-83e0-91c0b5336fc0.PNG)

@Raunaksingh100, did you hit the problem at running `npm run start` on a Windows platform? Did you use Powershell or Windows traditional CMD? If it's on Windows, try this sequence of commands instead of using `npm run start`:

```
npm run clean
SET ELEVENTY_ENV=development
npx eleventy --serve --port=3000 --config=eleventy.config.js
```

**Comment by Raunaksingh100** [comment]
> > hi i am facing this problem when i add this COMMAND "NPM RUN START" plz anyone tell me how i run this project in my local plz help,
> > see in the screenshot for the better understanding....
> > 
> > ![my issue with npm](https://user-images.githubusercontent.com/65535360/87951073-c161c680-cac5-11ea-83e0-91c0b5336fc0.PNG)
> 
> @Raunaksingh100, did you hit the problem at running `npm run start` on a Windows platform? Did you use Powershell or Windows traditional CMD? If it's on Windows, try this sequence of commands instead of using `npm run start`:
> 
> ```
> npm run clean
> SET ELEVENTY_ENV=development
> npx eleventy --serve --port=3000 --config=eleventy.config.js
> ```
> 

I add this all in which poweshell ?

npm run clean
> SET ELEVENTY_ENV=development
> npx eleventy --serve --port=3000 --config=eleventy.config.js
> ```

??

**Comment by Raunaksingh100** [comment]
> > hi i am facing this problem when i add this COMMAND "NPM RUN START" plz anyone tell me how i run this project in my local plz help,
> > see in the screenshot for the better understanding....
> > 
> > ![my issue with npm](https://user-images.githubusercontent.com/65535360/87951073-c161c680-cac5-11ea-83e0-91c0b5336fc0.PNG)
> 
> @Raunaksingh100, did you hit the problem at running `npm run start` on a Windows platform? Did you use Powershell or Windows traditional CMD? If it's on Windows, try this sequence of commands instead of using `npm run start`:
> 
> ```
> npm run clean
> SET ELEVENTY_ENV=development
> npx eleventy --serve --port=3000 --config=eleventy.config.js
> ```
> 


For npm start we have to change the start command  in package.json 

Is this correct or wrong ???

Or simpley  add this commands in poweshell
??

**Comment by cindyli** [comment]
> For npm start we have to change the start command in package.json
> 
> Is this correct or wrong ???
> 
> Or simpley add this commands in poweshell
> ??

You don't need to change `package.json`. Just run these commands in Powershell.

**Comment by Raunaksingh100** [comment]
thanks its works @cindyli thanks a lot

**Comment by TeddTech** [comment]
After implementing the design changes suggested in #322 to improve the visual appearance of the We Count website all of the contrast ratio issues mentioned still persist and now additionally includes the Tag Filter tags on Views page.

To reproduce this:
- Go to the [deploy preview](https://deploy-preview-334--wecount.netlify.app/) of PR #334 
- Navigate to the Views page
- Expand Tag Filters and select one or more tags
- Click "Apply Filters" button
- Run lighthouse 
- See referenced contrast ratio issues in the accessibility section of the lighthouse report

**Comment by cindyli** [comment]
@uttaraghodke, @lliskovoi: following @TeddTech's comment about the persistence of the contrast ratio problem, we discussed this in an email thread before and @jobara's research result gave a good summary:

```
In WCAG 2.1, it defines [the minimum contrast ratio](https://www.w3.org/TR/WCAG21/#contrast-minimum) to be 4.5:1 except for the case of Large Text (contrast ratio 3:1), incidental and logotypes.

**What is Large Text?** [Large scale text](https://www.w3.org/TR/WCAG21/#dfn-large-scale) is defined as 18 point or 14 point bold.

**What is a point (pt) on a digital device?** Well that’s confusing. (See: [wcag issue #181](https://github.com/w3c/wcag/issues/181)). That discussion lead to an update to [Understanding WCAG 2.1](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum) document; which uses the CSS calculation and is based off a 16px base font if I’m not mistaken. 1pt = 1.333px, 14pt ≈ 18.5px and 18pt ≈ 24px.
```

The cause of the problem still persists after the design change that increases the article font size from 16px to 18px for the desktop layout because the size of the large text in WCAG 2.1 spec is 24px. Moreover, the changed font size for the mobile layout is even smaller.

I wonder if it would be more reasonable to increase contrast ratios by picking dark font colors or lighter background colors.

**Comment by Raunaksingh100** [comment]
hi iam facing this problem in windows while comminting in your IDRC project see in the screnshot for the better undertanding
PLZ HELP ME  @cindyli @gtirloni 

![error in idrc](https://user-images.githubusercontent.com/65535360/88266700-41b64080-cced-11ea-9f42-830c36f7657e.PNG)

**Comment by cindyli** [comment]
At today's meeting, the link colors are decided to change to:

* Link text when the background is white: 5879A0
* Link text when background is yellow (tags and link hover/focus): 4D6B8D

**Comment by Raunaksingh100** [comment]
> At today's meeting, the link colors are decided to change to:
> 
> * Link text when the background is white: 5879A0
> * Link text when background is yellow (tags and link hover/focus): 4D6B8D
> 

Is there Gitter group of this organisation if then plz.share the link

**Comment by Raunaksingh100** [comment]
> At today's meeting, the link colors are decided to change to:
> 
> * Link text when the background is white: 5879A0
> * Link text when background is yellow (tags and link hover/focus): 4D6B8D
> 

I will do it , but this do only for 1 link or for all the link ??? See in the screenshot given above
